src/styles/banner.scss

```diff
@@ -1,81 +1,53 @@
 button {
-    background: #4ca1af;
-    width: 35%;
-    height: 3rem;
-    // padding: 0.8rem 0rem;
-    border-radius: 10px;
-    color: #fff;
-    cursor: pointer;
-    font-family: "Roboto", sans-serif;
-    font-size: 1.1rem;
-    letter-spacing: 0.08rem;
-    a {       //for NavLink component that creates an a tag.
-        text-decoration: none;
-        color: white;
-    }
-    &:hover {
-        transition: all 0.5s ease;
-        background: #37b95e;
-    }
-}
+  background: #4ca1af;
+  padding: 0.125rem 1.825rem;
+  border-radius: 10px;
+  color: #fff;
+  cursor: pointer;
+  font-family: 'Roboto', sans-serif;
+  font-size: 1.1rem;
+  letter-spacing: 0.08rem;
 
-h4 {
-    padding-top: 9rem;
-    font-size: 3rem;
-    text-align: center;
-    display: flex;
-    justify-content: center;
-    font-family: "Roboto", sans-serif;
-}
+  a {
+    text-decoration: none;
+    color: white;
+  }
 
-.latest {
-    text-align: center;
-    font-size: 1.5rem;
-    font-family: "Roboto," sans-serif
+  &:hover {
+    transition: all 0.5s ease;
+    background: #37b95e;
+  }
 }
 
-// tagline and banner section
-
 .banner {
-    height: 80vh;
-    min-height: 80vh;
```

Comment:
I rarely set a height on anything. Grid makes it possible, but usually isn't necessary and can still be a way to shoot yourself in the foot.